moveit/scripts/create_deprecated_headers.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Copyright 2021 PickNik Inc.
```

Comment:
2024, and you should feel free to either use your own name or your company's here.

Reply:
Oh awesome :smile:

Reply:
Done :+1: